### PR TITLE
Export internal Python SDK, document updating tags

### PR DESCRIPTION
**Pull request by arcticfly**
We're now exporting the internal SDKs from both our Python and TypeScript packages.

### CODE REVIEW

File: docs/features/updating-tags.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Running Inference"
```
Comment (corbt) [inline docs/features/updating-tags.mdx]: "Updating Tags"

Comment (corbt) [reply docs/features/updating-tags.mdx]: Actually looks like this file was copied over but retains the content on running inference?

**Review by corbt — APPROVED** [review]
Thanks, this looks good.

File: docs/features/updating-tags.mdx (hunk 1 above)
Comment (arcticfly) [reply docs/features/updating-tags.mdx]: Yep, I haven't written any of the docs yet

**Pull request merged**